dev/source/docs/learning-ardupilot-the-example-sketches.rst

```diff
@@ -53,6 +53,29 @@ connect to a Pixhawk on Linux:
 Using the --setup option puts mavproxy into raw serial mode, instead of
 processed MAVLink mode. That is what you need for the example sketches.
 
+Running sketches in SITL
```

Comment:
instead of 'sketches' I'd prefer 'examples'

Reply:
Fixed